qiskit/schemas/qobj_schema.json

```diff
@@ -639,7 +639,7 @@
                     },
                     "required": [
                         "label",
-                        "type"
+                        "snapshot_type"
```

Comment:
Should the `type` above this (line 636) also be updated to `snapshot_type`?

Reply:
Yes I think it should!